.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     env:
-      PHP_EXTENSIONS: dom, json, libxml, mbstring, pdo_sqlite, soap, xml, xmlwriter, :apcu, :imagick
+      PHP_EXTENSIONS: none, dom, json, libxml, mbstring, openssl, pdo_sqlite, soap, tokenizer, xml, xmlwriter
```

Comment:
Not sure why, but `windows-latest` requires `openssl`.

Reply:
https://github.com/sebastianbergmann/php-code-coverage/issues/992#issue-1644668197